content/en/service_management/service_level_objectives/_index.md

```diff
@@ -211,21 +213,35 @@ Each event includes the following SLO configuration information:
 
 Three types of SLO audit events appear in the Event Explorer:
 
-1. `SLO Created` events show all four pieces of SLO configuration information at creation time.
-2. `SLO Modified` events show a what configuration information changed during a modification
-3. `SLO Deleted` events show all four pieces of configuration information the SLO had right before it was deleted
+- `SLO Created` events show the SLO configuration information at creation time
+- `SLO Modified` events show what configuration information changed during a modification
+- `SLO Deleted` events show the configuration information the SLO had before it was deleted
+
+**SLO status correction audit events:**
+
+Each event includes the following SLO status correction configuration information:
 
-To get a full list of all SLO audit events, enter the search query `tags:audit,slo` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO.
+- SLO Name
+- Status correction start and end times and timezone
+- Status correction category
```

Comment:
```suggestion
#### Status correction audit event categories
```

Reply:
This isn't a separate section, it's part of the list of the pieces of info in the event